src/components/CytoscapeGraph/CytoscapeContextMenu.tsx

```diff
@@ -18,6 +20,8 @@ type ContextMenuContainer = HTMLDivElement & {
 type ContextMenuProps = {
   element: any;
   contextMenu: TippyInstance;
+  jaegerIntegration: boolean;
+  jaegerURL: string;
```

Comment:
I think we can move these to the `NodeContextMenu` and take these values from redux (they are in redux, right?) If not, use a HOC in the GraphPage, to keep this component a tad more generic.
I can do that if you are OK with it.